mindmeld/converter/converter.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains the abstract Coverter class used to convert other software's
+projects into mindmeld projects"""
+
+from abc import ABC, abstractmethod
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Converter(ABC):
+
+    def __init__(self):
+        pass
+
+    @abstractmethod
+    def convert_project(self):
+        pass
+
+    @abstractmethod
+    def create_mindmeld_directory(self):
+        pass
+
+    @abstractmethod
+    def create_training_data(self):
```

Comment:
Could we rename this and the following method to `create_mindmeld_training_data` and `create_mindmeld_init` ? I think that's a bit more clear, and is consistent with `create_mindmeld_directory` above

Reply:
Yes, I can change that.